js/ui/draggable.js

```diff
@@ -866,7 +866,7 @@ const Draggable = DOMComponent.inherit({
         const $targetDraggableElement = this.$element();
 
         const mousePosition = getMousePosition(e);
-        const elements = domAdapter.elementsFromPoint(mousePosition.x, mousePosition.y);
+        const elements = domAdapter.elementsFromPoint(mousePosition.x, mousePosition.y, this.element());
```

Comment:
Does this work? Seem like it gonna be
```suggestion
        const elements = domAdapter.elementsFromPoint(mousePosition.x, mousePosition.y, this.$element().get(0));
```

Reply:
both these approaches work